app/src/main/AndroidManifest.xml

```diff
@@ -1,66 +1,83 @@
 
     <application
-            android:allowBackup="true"
-            android:icon="@mipmap/ic_launcher"
-            android:label="@string/app_name"
-            android:roundIcon="@mipmap/ic_launcher_round"
-            android:supportsRtl="true"
-            android:name=".FenixApplication"
-            android:theme="@style/NormalTheme"
-            android:usesCleartextTraffic="true"
-            tools:ignore="UnusedAttribute">
-        <activity android:name=".HomeActivity"
-                  android:launchMode="singleTask"
-                  android:windowSoftInputMode="adjustResize"
-                  android:configChanges="keyboard|keyboardHidden|mcc|mnc|orientation|screenSize|locale|layoutDirection|smallestScreenSize|screenLayout">
+        android:name=".FenixApplication"
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/NormalTheme"
+        android:usesCleartextTraffic="true"
+        tools:ignore="UnusedAttribute">
+        <activity
+            android:name=".HomeActivity"
+            android:configChanges="keyboard|keyboardHidden|mcc|mnc|orientation|screenSize|locale|layoutDirection|smallestScreenSize|screenLayout"
+            android:launchMode="singleTask"
+            android:windowSoftInputMode="adjustResize">
             <intent-filter>
-                <action android:name="android.intent.action.MAIN"/>
+                <action android:name="android.intent.action.MAIN" />
 
-                <category android:name="android.intent.category.LAUNCHER"/>
+                <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
 
-        <activity android:name=".customtabs.CustomTabActivity"
-                  android:configChanges="keyboard|keyboardHidden|mcc|mnc|orientation|screenSize|locale|layoutDirection|smallestScreenSize|screenLayout"
-                  android:windowSoftInputMode="adjustResize|stateAlwaysHidden"
-                  android:exported="false"
-                  android:taskAffinity=""
-                  android:persistableMode="persistNever"
-                  android:autoRemoveFromRecents="false"
-                  android:label="@string/app_name" />
+        <activity
+            android:name=".customtabs.CustomTabActivity"
+            android:autoRemoveFromRecents="false"
+            android:configChanges="keyboard|keyboardHidden|mcc|mnc|orientation|screenSize|locale|layoutDirection|smallestScreenSize|screenLayout"
+            android:exported="false"
+            android:label="@string/app_name"
+            android:persistableMode="persistNever"
+            android:taskAffinity=""
+            android:windowSoftInputMode="adjustResize|stateAlwaysHidden" />
+
+        <activity
```

Comment:
new activity here copied from CustomTabActivity

Reply:
That persistableMode attribute is a no-op unless it's the task root activity, which it shouldn't be here. Setting the label to the app_name is really a no-op, as it's the default label if it's set on the application.

Reply:
Nice! Okay took those out